### PR TITLE
Cut v1.5.0

### PR DESCRIPTION
**Pull request by beorn7**
Also @gotjosh , who did the API change.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 1.5.0 / 2020-03-03
+
+* [FEATURE] promauto: Add a factory to allow autoregistration with a local registry. #713
```
Comment (bwplotka) [inline CHANGELOG.md]:
```suggestion
* [FEATURE] promauto: Add a factory to allow auto registration with a local registry. #713
```
:thinking:

Comment (beorn7) [reply CHANGELOG.md]: In the doc comment, we use `auto-register` once, but the proper way is "automatic registration", which I'll do now.

Comment (bwplotka) [reply CHANGELOG.md]: Sorry for being picky ^^ I learned from the best.... (((:

**Comment by bwplotka** [comment]
:+1:

**Pull request merged**